src/Aliencube.AzureFunctions.Extensions.OpenApi.Core/IDocument.cs

```diff
@@ -23,6 +23,11 @@ namespace Aliencube.AzureFunctions.Extensions.OpenApi.Core.Abstractions
     /// </summary>
     public interface IDocument
     {
+        /// <summary>
+        /// Gets the underlying <see cref="OpenApiDocument"/>.
+        /// </summary>
+        OpenApiDocument OpenApiDocument { get; }
```

Comment:
@gsteendp I'm interested in this idea. But I haven't seen an actual use case of this to be exposed as a public property. Would you please give an example?

Reply:
It allows the consumer of the library to modify / verify the underlying model before rendering in case the wrapper IDocument does not cover a scenario for a particular implementation. (For example, customization that the IDocumentHelper methods don't allow you to do) It also allows you to test the object graph before rendering in the library without using reflection.

Reply:
@gsteendp We probably should separate this part from this PR.

Reply:
Reverted the security requirements aspect from this pull

Reply:
Reverted the security requirements aspect from this pull